src/schema/metadata/Columns.yaml

```diff
@@ -0,0 +1,8 @@
+---
+key_name: Columns
+description: |
+  Names of columns in file.
+type: array
+items:
+  type: string
+  minLength: 1
```

Comment:
Could we just assume that strings have a minimum length of 1? If the field is provided with a string, then I think it should have at least one character to be valid, from a BIDS standpoint. Missing data should be provided with `"n/a"`, so there's not really a case for `""` appearing in jsons.

Reply:
agreed! And then users can still abuse it by providing `" "`, but it might be sufficient to make them think twice :-)

Reply:
I think the same can apply to `minItems` when the type is `array`, so I'll remove all cases of `minLength: 1` and `minItems: 1` from the YAML files.